tests/functional/lib/mesh/climesh.go

```diff
@@ -150,6 +150,111 @@ func (m *CLIMesh) Nodes() map[string]Node {
 	return nodes
 }
 
+func populateNodedefConnection(meshDefinition *YamlData, existingMesh *CLIMesh) error {
```

Comment:
let's rename this to `createNodedefConnections`

let's also keep `connections` plural everywhere, so `NodedefConnections` instead of `NodedefConnection`

Reply:
I have made the required change